addons/mail_bot/static/src/messaging/component/messaging_menu/messaging_menu_tests.js

```diff
@@ -5,35 +5,22 @@ const {
     afterEach: utilsAfterEach,
     afterNextRender,
     beforeEach: utilsBeforeEach,
-    getServices,
     pause,
     start: utilsStart,
 } = require('mail.messaging.testUtils');
 
-const MailBotService = require('mail_bot.MailBotService');
-
-const { makeTestPromise } = require('web.test_utils');
-
 QUnit.module('mail_bot', {}, function () {
 QUnit.module('messaging', {}, function () {
 QUnit.module('component', {}, function () {
 QUnit.module('MessagingMenu', {
     beforeEach: function () {
         utilsBeforeEach(this);
         this.start = async params => {
-            const services = Object.assign({}, getServices(), {
-                mailbot_service: MailBotService,
-            });
-
-            if (this.widget) {
-                this.widget.destroy();
-            }
-            let { discussWidget, widget } = await utilsStart(Object.assign({}, params, {
+            let { env, widget } = await utilsStart(Object.assign({}, params, {
                 data: this.data,
                 hasMessagingMenu: true,
-                services,
             }));
-            this.discussWidget = discussWidget;
+            this.env = env;
```

Comment:
Then I think this is the env I meant that was not used in the file from yesterday.